### ISSUE TITLE
hosting binaries

### ISSUE THREAD

**Issue by DonatJR** [DonatJR/adbackup#7]
should we maybe host binaries somewhere when we are nearing 1.0.0 (or something like that) of the cli tool so potential users don't have to build them themselves?

**Comment by SailReal** [comment]
afaik github directly supports hosting binaries...nice idea and should we do! :+1:

**Comment by SailReal** [comment]
Yeah, we can use github which handles releases pretty well: https://help.github.com/articles/creating-releases/ (upload binaries and assign them to an branch/commit)

**Comment by DonatJR** [comment]
should we maybe wait with this until we have a 'complete' version (i.e. 1.0.0)?
